src/App.jsx

```diff
@@ -14,57 +15,62 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () =>{
+  const [selectedGood, setSelectedGood] = useState('Jam');
+
+  function handleSelectedGood(good){
+    const selectedGood = goods.find((item) => item === good)
```

Comment:
The variable `selectedGood` is redeclared here, which shadows the state variable `selectedGood` from the component's state. Consider renaming this variable to avoid confusion and potential bugs.